src/main.cpp

```diff
@@ -76,8 +78,21 @@ class MetaGenerationConsumer : public clang::ASTConsumer {
         // Generate TypeScript definitions
         if (!cla_outputDtsFolder.empty()) {
             llvm::sys::fs::create_directories(cla_outputDtsFolder);
+
+            // find iOS SDK docset path
+            std::string docSetPath;
+            if (!cla_docSetFile.empty()) {
+                docSetPath = cla_docSetFile.getValue();
+            }
+            else if (struct passwd* pwd = getpwuid(getuid())) {
```

Comment:
@fealebenpae  This makes sense but I didn't manage to find a platform specific docset package. Also, Xcode doesn't give you the option to choose between iOS doc for device and iOS doc for emulator. Also, the name of the docset package doesn't hint a specific platform, so I came to conclusion that the docset package is the same for device and simulator. Am I missing something?